foreshadow/tests/test_preprocessor.py

```diff
@@ -203,7 +203,8 @@ def test_preprocessor_init_json_pipeline_map():
     transformer = proc._pipeline_map["crim"].steps[0][PipelineStep["CLASS"]]
 
     assert type(transformer).__name__ == "StandardScaler"
-    assert hasattr(transformer, "name")
+    # assert hasattr(transformer, "name")
+    # TODO when this test is replaced, add the new test for name attribute.
```

Comment:
Do we want to throw up an issue for this?

Reply:
I think it's obvious from the removal of preprocessor with DataPreparer, was just a reminder that the name attribute is a component to be tested

Reply:
👍